gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/compile/ResultSetNode.java

```diff
@@ -1090,8 +1091,13 @@ public void renameGeneratedResultNames() throws StandardException
 		for (int i=0; i<resultColumns.size(); i++)
 		{
 			ResultColumn rc = (ResultColumn) resultColumns.elementAt(i);
-			if (rc.isNameGenerated())
-				rc.setName(Integer.toString(i+1));
+			if (rc.isNameGenerated()) {
+				if (!StringUtils.isBlank(System.getenv("SNAPPY_HOME"))) {
+					rc.setName("_c" + Integer.toString(i));
```

Comment:
The presence of "SNAPPY_HOME" environment variable may not be guaranteed especially if starting programatically -- use "Misc.getMemStore().isSnappyStore()". That's the standard pattern in product (except in launcher where product may not yet have started).